### PR TITLE
Hotfix/role-cycle-2023

### PR DESCRIPTION
**Pull request by YoanRouleau**
Tout est dans le titre :)

### CODE REVIEW

File: config.json.example [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "prefix": "",
```
Comment (BlueskyFR) [inline config.json.example]: that should be `"prefix": "!",` IMO

**Review by BlueskyFR — CHANGES_REQUESTED** [review]

File: config.json.example (hunk 1 above)
Comment (YoanRouleau) [reply config.json.example]: I'd say that even the prefix field must remain hidden, so no one knows that "!" is used as the command prefix.

Comment (BlueskyFR) [reply config.json.example]: why should it be hidden 😂

Comment (YoanRouleau) [reply config.json.example]: Bon comme tu veux

**Review by BlueskyFR — APPROVED** [review]

**Pull request merged**